frontend/src/app/features.ts

```diff
@@ -0,0 +1,59 @@
+type FeatureHelpInfo = {
```

Comment:
The purpose if this file is more or less clear - leaning towards less - as we flesh out the other features, let's add some comments to explain what this file does.